ci/terraform/oidc/authorize.tf

```diff
@@ -77,7 +77,7 @@ module "authorize" {
     local.authentication_security_group_id,
     local.authentication_oidc_redis_security_group_id,
   ]
-  subnet_id                              = local.authentication_subnet_ids
+  subnet_id                              = local.authentication_protected_subnet_ids
```

Comment:
We want to roll this out per environment

Reply:
Have added a flag for this